### PR TITLE
Security vulnerability fix

### PR DESCRIPTION
**Pull request by nitishsingla1234**
### JIRA link (if applicable) ###
NA

### Change description ###
Fixing the security vulnerability introduced by tomcat embedded jar.

#### If you're adding a new secret then do the following ####
Run the script in `./bin/set-secret-in-all-vaults <microservice-name>` and paste the output below
This will ensure the secret is in all the vaults it needs to be

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
   id 'org.owasp.dependencycheck' version '5.3.2.1'
   id 'org.sonarqube' version '2.8'
   id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-  id 'org.springframework.boot' version '2.2.7.RELEASE'
+  id 'org.springframework.boot' version '2.3.0.RELEASE'
```
Comment (luigibk) [inline build.gradle:9]: This breaks the health checks library at the moment + fixes and adding a secret are really operations that belong to separate PRs

Comment (nitishsingla1234) [reply build.gradle:9]: Hi @luigibk Let me separate out the PR's
